util/util.go

```diff
@@ -86,19 +87,28 @@ func colorPrint(state check.State, s string) {
 }
 
 // prettyPrint outputs the results to stdout in human-readable format
-func PrettyPrint(r *check.Controls, summary check.Summary) {
+func PrettyPrint(r *check.Controls, summary check.Summary, noRemediations bool) {
 	colorPrint(check.INFO, fmt.Sprintf("%s %s\n", r.ID, r.Description))
 	for _, g := range r.Groups {
 		colorPrint(check.INFO, fmt.Sprintf("%s %s\n", g.ID, g.Description))
 		for _, c := range g.Checks {
 			colorPrint(c.State, fmt.Sprintf("%s %s\n", c.ID, c.Description))
+
+			if c.State == check.FAIL {
+				if len(c.ActualValue) == 0 {
+					fmt.Println("\t", c.FailedDescription)
+				} else {
+					PrintAttributes(c.ActualValue)
+				}
+				fmt.Println()
+			}
 		}
 	}
 
 	fmt.Println()
 
 	// Print remediations.
-	if summary.Fail > 0 || summary.Warn > 0 {
+	if !noRemediations && (summary.Fail > 0 || summary.Warn > 0) {
```

Comment:
I agree, the only reason this variable has negative meaning is because the app (docker-bench) has a "no remediations" flag, so it felt right to keep the variable as is. I'l change it because the bench-common doesn't need to be affected by the apps that are using it.